src/sentry/static/sentry/app/components/events/eventEntries.jsx

```diff
@@ -67,6 +68,25 @@ const EventEntries = createReactClass({
     };
   },
 
+  recordSourcemapError(errorType) {
+    analytics('sourcemap.sourcemap_error', {
+      'org_id': this.props.project.organization.id,
+      'group': this.props.event.groupID,
+      'error_type': errorType
+    })
+  },
+
+  componentDidMount() {
+    if (typeof this.props.event.errors !== 'undefined') {
+      let errorTypes = this.props.event.errors;
+      let errorType = [];
+      for (let i = 0; i < errorTypes.length; i++) {
+        errorType.push(errorTypes[i].type);
```

Comment:
a cool thing you can do here is take advantage of `.map()` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map to populate your array for you

something like `let x = errorTypes.map((errorType) => errorType.type)`